pkg/kernelmoduleloader/loader.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
Can we also rename this file from `loader.go` to `kernel_module_loader.go`?

We will more than likely have one for `bpf_probe_loader.go` in the future